source/users/sessions.txt

```diff
@@ -0,0 +1,488 @@
+desired expiration time.
+
+.. code-block:: bash
+
+   curl -X POST \
+     https://realm.mongodb.com/api/admin/v3.0/groups/{groupId}/apps/{appId}/security/refresh_token_expiration \
+     --data-raw '{
+       "expiration_time_seconds": 864000
+     }'
+
+If you successfully update the refresh token expiration time, the
+endpoint returns a ``204`` response.
+
+Verify & Decode a User Access Token
+-----------------------------------
+
+You can decode an access token to verify that it has the correct format
+and has a valid signature. The decoded response includes other
+information like when the access token expires.
+
+Call your App's :admin-api-endpoint:`Verify & decode an access token
+<operation/adminVerifyUserAccessToken>` Admin API endpoint with the
+token in the ``token`` field of the request body.
+
+.. io-code-block::
+
+   .. input::
+      :language: bash
+
+      curl -X POST \
+        https://realm.mongodb.com/api/admin/v3.0/groups/{groupId}/apps/{appId}/users/verify_token \
+        --data-raw '{
+          "token": "eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJiYWFzX2RldmljZV9pZCI6IjAwMDAwMDAwMDAwMDAwMDAwMDAwMDAwMCIsImJhYXNfZG9tYWluX2lkIjoiNWNkYjEyNDA4ZTIzMmFjNGY5NTg3ZmU4IiwiZXhwIjoxNjc2NTExMjgyLCJpYXQiOjE2NzY1MDk0ODIsImlzcyI6IjYzZWQ4MTJhNDNiZTcyYzE3NmFhNWQyMyIsInN0aXRjaF9kZXZJZCI6IjAwMDAwMDAwMDAwMDAwMDAwMDAwMDAwMCIsInN0aXRjaF9kb21haW5JZCI6IjVjZGIxMjQwOGUyMzJhYzRmOTU4N2ZlOCIsInN1YiI6IjYzZWQ4MTJhNDNiZTcyYzE3NmFhNWQyMSIsInR5cCI6ImFjY2VzcyJ9.7kHO9wjWvIaD3VewDyPhLyb-oRc7wTYZdD9-hroF-H4"
+        }'
+
+   .. output::
+      :language: json
+
+      {
+      	"sub": "63ed812a43be72c176aa5d21",
+      	"exp": 1676511282,
+      	"iat": 1676509482,
+      	"iss": "63ed812a43be72c176aa5d23",
+      	"domain_id": "5cdb12408e232ac4f9587fe8",
+      	"device_id": "000000000000000000000000"
+      }
+
+If the token is valid, you'll receive a ``200`` response.
+
+If the token is valid but expired, the response indicates that the token
+is expired rather than containing the decoded JWT.
+
+.. code-block:: json
+
+   "token expired"
+
+If the token is invalid, you'll receive a ``401`` response that contains
+an error message.
+
+.. code-block:: json
+
+   {
+     "error": "signature is invalid"
+   }
+
+End a User Session
+------------------
+
+Once a session has been established you cannot end it individually,
+though you can :ref:`revoke all of a user's active sessions
+<revoke-user-sessions>`.
+
+You can effectively end a user session by deleting all copies of the
+access token and refresh token. This prevents further access to the
+session and requires the user to authenticate and start a new session
+to continue.
+
+Each Realm SDK has a ``User.logOut()`` method that deletes local copies
+of the access token and refresh token and invalidates the refresh token
+so that it can't be used to get a new access token.
+
+If a user account is deleted, either by an administrator or by the user,
+then all of the user's sessions are automatically revoked.
```

Comment:
in this paragraph, consider ref to https://docs-atlas-staging.mongodb.com/atlas-app-services/docsworker-xlarge/get-access-token/users/manage/#delete-a-user